lib/orchestrator_client/config.rb

```diff
@@ -98,7 +98,8 @@ def load_token
     else
       validate_file('token-file', config['token-file'])
       token = File.open(config['token-file']) { |f| f.read.strip }
-      if token != URI.escape(token)
+      # If the token file contains illegal characters
+      if token =~ URI::UNSAFE
```

Comment:
As a security practice, we know the format of the token file. We could specifically check it only contained expected characters.

In this instance, the current check is probably sufficient.